plugins/woocommerce-blocks/tests/utils/integration/editor.jsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import { act, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { useState, useEffect } from '@wordpress/element';
+import {
+	BlockEditorKeyboardShortcuts,
+	BlockEditorProvider,
+	BlockList,
+	BlockTools,
+	BlockInspector,
+	WritingFlow,
+	ObserveTyping,
+} from '@wordpress/block-editor';
+import { Popover, SlotFillProvider } from '@wordpress/components';
+import { registerCoreBlocks } from '@wordpress/block-library';
+import { ShortcutProvider } from '@wordpress/keyboard-shortcuts';
+import {
+	createBlock,
+	unregisterBlockType,
+	getBlockTypes,
+} from '@wordpress/blocks';
+import '@wordpress/format-library';
+
+/**
+ * Internal dependencies
+ */
+import { waitForStoreResolvers } from './wait-for-store-resolvers';
+
+// Polyfill for String.prototype.replaceAll until CI is runnig Node 15 or higher.
```

Comment:
After reading this code, I realized that we aren't introducing any CI action for Integration tests. If we don't wanna add it in this PR, then probably we can add it in follow up PR assuming we would live to have a CI job for this. 

But in any case, I think we can remove this polyfill and use a node version higher than 15. What do you think?